parm/aero/variational/3dvar_gfs_aero_bump.yaml

```diff
@@ -0,0 +1,42 @@
+cost function:
+  cost type: 3D-Var
+  window begin: '$(AERO_WINDOW_BEGIN)'
+  window length: $(AERO_WINDOW_LENGTH)
+  analysis variables: &3dvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
+  geometry: $(GEOM_BKG)
+  background:
+    datapath: $(BKG_DIR)
+    filetype: fms restart
+    datetime: $(BKG_ISOTIME)
+    filename_core: $(BKG_YYYYmmddHHMMSS).fv_core.res.nc
+    filename_trcr: $(BKG_YYYYmmddHHMMSS).fv_tracer.res.nc
+    filename_cplr: $(BKG_YYYYmmddHHMMSS).coupler.res
+    state variables: [t,delp,sphum,sulf,bc1,bc2,oc1,oc2,
+                      dust1,dust2,dust3,dust4,dust5,
+                      seas1,seas2,seas3,seas4] 
+  background error: $<< $(BERROR_YAML)
+  observations: $<< $(OBS_LIST)
+variational:
+  minimizer:
+    algorithm: DRIPCG
+  iterations:
+  - ninner: 35 
+    gradient norm reduction: 1e-10
+    test: on
+    geometry: $(GEOM_ANL)
+    diagnostics:
+      departures: bkgmob
```

Comment:
Note to self/reader: there is an OOPS PR to "flip" this so that it will actually be `ombg`

Reply:
Should I change anything?

Reply:
for now, no